cut_copy_paste_tests.py

```diff
@@ -244,7 +244,7 @@ def read_list_of_commands():
 
         def skip_comments():
             nonlocal pos
-            while tests[pos] == '[':
+            while tests[pos] == '[': # [

```

Comment:
What are these comments for?

Reply:
To balance opening and closing brackets.